lib/tools/common/patching_parallel.py

```
@@ -0,0 +1,1244 @@
+
+    finally:
+        # Clean up overlayfs mounts
+        log.info("Cleaning up overlayfs mounts...")
+        for overlay in mounts:
+            overlay.unmount()
+
+        # Remove temp directory
+        try:
+            if 'temp_base' in locals():
+                shutil.rmtree(temp_base)
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)